dbt/include/redshift/macros/materializations/materialized_view.sql

```diff
@@ -1,106 +1,5 @@
-{% macro redshift__get_alter_materialized_view_as_sql(
-    relation,
-    configuration_changes,
-    sql,
-    existing_relation,
-    backup_relation,
-    intermediate_relation
-) %}
-
-    -- apply a full refresh immediately if needed
-    {% if configuration_changes.requires_full_refresh %}
-
-        {{ get_replace_materialized_view_as_sql(relation, sql, existing_relation, backup_relation, intermediate_relation) }}
-
-    -- otherwise apply individual changes as needed
-    {% else %}
-
-        {%- set autorefresh = configuration_changes.autorefresh -%}
-        {%- if autorefresh -%}{{- log('Applying UPDATE AUTOREFRESH to: ' ~ relation) -}}{%- endif -%}
-
-        alter materialized view {{ relation }}
-            auto refresh {% if autorefresh.context %}yes{% else %}no{% endif %}
-
-    {%- endif -%}
-
-{% endmacro %}
-
-
-{% macro redshift__get_create_materialized_view_as_sql(relation, sql) %}
-
-    {%- set materialized_view = relation.from_runtime_config(config) -%}
-
-    create materialized view {{ materialized_view.path }}
-        backup {% if materialized_view.backup %}yes{% else %}no{% endif %}
-        diststyle {{ materialized_view.dist.diststyle }}
-        {% if materialized_view.dist.distkey %}distkey ({{ materialized_view.dist.distkey }}){% endif %}
-        {% if materialized_view.sort.sortkey %}sortkey ({{ ','.join(materialized_view.sort.sortkey) }}){% endif %}
-        auto refresh {% if materialized_view.autorefresh %}yes{% else %}no{% endif %}
-    as (
-        {{ materialized_view.query }}
-    );
-
-{% endmacro %}
-
-
-{% macro redshift__get_replace_materialized_view_as_sql(relation, sql, existing_relation, backup_relation, intermediate_relation) %}
-    {{ redshift__get_drop_relation_sql(existing_relation) }};
-    {{ get_create_materialized_view_as_sql(relation, sql) }}
-{% endmacro %}
-
-
 {% macro redshift__get_materialized_view_configuration_changes(existing_relation, new_config) %}
     {% set _existing_materialized_view = redshift__describe_materialized_view(existing_relation) %}
     {% set _configuration_changes = existing_relation.materialized_view_config_changeset(_existing_materialized_view, new_config) %}
     {% do return(_configuration_changes) %}
 {% endmacro %}
```

Comment:
why does this macro remain here in `adapters.sql`?

Reply:
whoops i read the file heading wrong. i see it's actually in `dbt/include/redshift/macros/materializations/materialized_view.sql` not `adapters.sql`